cogs/send_introduction_reminders.py

```diff
@@ -134,31 +134,30 @@ async def send_introduction_reminders(self) -> None:
                     if message_contains_opt_in_out_button:
                         await message.edit(view=None)
 
-                if member in guild.members:
-                    await member.send(
-                        content=(
-                            "Hey! It seems like you joined "
-                            f"the {self.bot.group_short_name} Discord server "
-                            "but have not yet introduced yourself.\n"
-                            "You will only get access to the rest of the server after sending "
-                            "an introduction message."
-                        ),
-                        view=(
-                            self.OptOutIntroductionRemindersView(self.bot)
-                            if settings["SEND_INTRODUCTION_REMINDERS"] == "interval"
-                            else None  # type: ignore[arg-type]
-                        ),
-                    )
-
-                    await SentOneOffIntroductionReminderMember.objects.acreate(
-                            member_id=member.id
-                        )
-                else:
+                if member not in guild.members:
                     logger.info(
                         "Member: {member} left the server before the introduction "
```

Comment:
Value formatting inside of log messages should use the % syntax. See [line 92](https://github.com/CSSUoB/TeX-Bot-Py-V2/blob/745f1bb33f1e6a9922e0bfb4fecd5e75c8d5e33e/cogs/send_introduction_reminders.py#L92) for an example.

```python
logger.info(
    (
        "Member with ID: %s does not need to be sent a reminder "
        "because they have left the server."
    ),
    member.id,
)
```